docs/quickstarts/nextjs.mdx

```diff
@@ -277,83 +268,116 @@ export default function Example() {
 
-```tsx filename="pages/api/auth.ts"
-import type { NextApiRequest, NextApiResponse } from "next";
-import { getAuth } from "@clerk/nextjs/server";
 
-export default function handler(req: NextApiRequest, res: NextApiResponse) {
-  const { userId } = getAuth(req);
-  if (!userId) {
-    res.status(401).json({ error: "Unauthorized" });
-    return;
-  }
-  // retrieve data from your database
-  res.status(200).json({});
-}
-```
+The [`auth()`][auth-ref] helper will return the [`Authentication`](/docs/references/nextjs/authentication-object) object of the currently active user. Now that request data is available in the global scope through Next.js's `headers()` and `cookies()` methods, passing the request object to Clerk is no longer required.
 
-In some cases, you may need the full [`User`](/docs/references/javascript/user/user) object. For example, if you want to access the user's email address or name. You can use the [`clerkClient`](/docs/references/backend/overview) helper to get the full `User` object.
+The [`currentUser()`][currentuser-ref] helper will return the [`User`][user-object] object of the currently active user. This is helpful if you want to render information, like their first and last name, directly from the server.
 
+<Callout type="info">
+  Any requests from a Client Component to a Route Handler will automatically include the user's current token as the Bearer token.
+</Callout>
 
-```tsx filename="pages/api/auth.ts"
-import { clerkClient } from "@clerk/nextjs";
-import { getAuth } from "@clerk/nextjs/server";
-import type { NextApiRequest, NextApiResponse } from "next";
+<Tabs items={["Route Handler", "Route Handler w/ User Fetch"]}>
+  <Tab>
+    You can protect your API routes by using the [`auth()`](/docs/references/nextjs/auth) helper, if the route is not protected by [middleware](/docs/references/nextjs/auth-middleware). 
 
-export default async function handler(
-  req: NextApiRequest,
-  res: NextApiResponse
-) {
-  const { userId } = getAuth(req);
+    ```tsx filename="app/api/user/route.[jsx/tsx]"
+    import { NextResponse } from 'next/server';
+    import { auth } from '@clerk/nextjs';
 
-  if (!userId) {
-    return res.status(401).json({ error: "Unauthorized" });
-  }
+    export async function GET() {
 
-  const user = userId ? await clerkClient.users.getUser(userId) : null;
+      const { userId } = auth();
 
-  // use the user object to decide what data to return
+      if (!userId) {
+        return new NextResponse("Unauthorized", { status: 401 });
+      }
 
-  return res.status(200).json({});
-}
-```
+      // Perform your Route Handler's logic
 
-### Server side render user data
+      return NextResponse.json({ userId }, { status: 200 });
+    }
+    ```
+  </Tab>
+  <Tab>
+    You can protect your API routes by using the [`auth()`](/docs/references/nextjs/auth) helper, if the route is not protected by [middleware](/docs/references/nextjs/auth-middleware). 

```

Comment:
We also need to update this wording to be accurate, as `auth()` relies on the middleware executing.